modules/meta.analysis/NAMESPACE

```diff
@@ -1,7 +1,9 @@
 # Generated by roxygen2: do not edit by hand
 
 export(approx.posterior)
+export(assign.treatments)
```

Comment:
Nit: Would it make sense to rename this `assign_treatments` as long as we're moving it, to avoid any confusion from looking like as S3 method? (Not that `base::assign` seems all that likely to become a generic...)